rust/cubesql/cubesql/src/compile/mod.rs

```diff
@@ -20132,4 +20132,50 @@ LIMIT {{ limit }}{% endif %}"#.to_string(),
         assert!(sql.contains(" AS DOUBLE PRECISION)"));
         assert!(sql.contains(" AS DECIMAL(38,10))"));
     }
+
+    #[tokio::test]
```

Comment:
Let's put this test in `tests` folder, it can be called `push_down` or something.